python_tests/test_csv_download.py

```diff
@@ -5,11 +5,16 @@
 import optuna
 from optuna.trial import TrialState
 from optuna_dashboard._app import create_app
+from packaging import version
 import pytest
 
 from .wsgi_client import send_request
 
 
+if version.parse(optuna.__version__) < version.parse("3.4.0"):
+    pytest.skip("Study.metrics_name is introduced at v3.4.0", allow_module_level=True)

```

Comment:
[fyi] This dependency should be fixed, but it looks out of the scope of this PR.

https://github.com/optuna/optuna-dashboard/pull/723/files#diff-0f7b622fa29613e91a846a5635ff2339348136638848391f330189b81b224a80R471-R472